src/views/ProductView.vue

```diff
@@ -469,7 +470,7 @@ const product = asyncComputed(
  * has not been loaded yet or could not be loaded.
  */
 const productIsHidden = computed(() => {
-    if (product.value == null) {
+    if (product.value === null) {
```

Comment:
I personally would not use a strict equality check here, because if for whatever reason an undefined makes its way here in the future, you've got a hard to debug error

Reply:
the same goes for all other `=== null`

Reply:
That makes sense, I will change it.

Reply:
080154f5fc54179aab329cae89d31569cc0d5945